_commands/editors/vi.md

```diff
@@ -0,0 +1,39 @@
+vi
+
+
+
+'vi', also known as "vim" (Vi IMproved), is a text editor for programmers, designed to facilitate optimal efficiency when writing code.
```

Comment:
`vi != vim`. Vim has more features than Vi. However, some Linux distributions will do things like provide a `vi` executable that is actually Vim. I'd like to see this addressed in the document. See for example http://askubuntu.com/a/6503/136133